doc/performance-investigations/0022-upgrade-cloud-run-execution-environment-gen2/summary.md

```diff
@@ -0,0 +1,115 @@
+
+## Benchmark profile
+
+| Option                 | value                               |
+|------------------------|-------------------------------------|
+| Requests file          | test_business_benchmark_pdfkit.json |
+| Run time               | 20m                                 |
+| User wait time minimum | 1s                                  |
+ | User wait time maximum | 2s                                  |
+| Clients                | 50                                  |
+| Hatch rate             | 50                                  |
+
+## Results
+
+### Execution Environment Performance
+
+#### Results based on 99th percentile timings.
+
+| Metric                                   | Baseline (gen1)                                                                    | Investigation (gen2)                                                                    |
+|------------------------------------------|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| **GET** <br/>(*/submitted/download-pdf*) | 1900ms                                                                             | 1600ms                                                                                  |
+| Total Requests                           | 2,176                                                                              | 2,146                                                                                   |
+| 99th percentile Max CPU Usage (%)        | 30.9%                                                                              | 29.9%                                                                                   |
+| Total Failures                           | 0                                                                                  | 0                                                                                       |
+| Error Percentage                         | 0.0%                                                                               | 0.0%                                                                                    |
+| Results                                  | [benchmark_results](outputs/baseline/2023-02-27/Results_For_Latest_Benchmark.yaml) | [benchmark_results](outputs/investigation/2023-02-27/Results_For_Latest_Benchmark.yaml) |
+| Performance Graph                        | [result](outputs/baseline/2023-02-27/performance_graph.png)                        | [result](outputs/investigation/2023-02-27/performance_graph.png)                        |
+
+#### Detailed Results.
+
+| Execution Environment | Type | Name                    | Request Count | Failure Count | Average Response Time | Average Content Size | Requests/s | 50%  | 66%  | 75%  | 80%  | 90%  | 95%  | 98%  | 99%  | 99.90% | 99.99% | 100% | outputs                                     |
```

Comment:
Minor detail, could we add the units for `average response time` and `average content size`

Reply:
👍